src/anomalib/metrics/per_image/__init__.py

```diff
@@ -7,8 +7,7 @@
 # Copyright (C) 2024 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-from .binclf_curve import BinclfThreshsChoice
-from .enums import StatsOutliersPolicy, StatsRepeatedPolicy
+from .enums import BinclfThreshsChoice, StatsOutliersPolicy, StatsRepeatedPolicy
```

Comment:
maybe in the future PRs, can we think about a more descriptive name for `BinclfThreshsChoice`